.github/contributing-core-collabs-team.md

```diff
@@ -0,0 +1,27 @@
+### The Core Collaborators team
+[Core collabs]: #the-core-collaborators-team
+
+The Core Contributors team consists of one member of Umbraco HQ, [Sebastiaan][Sebastiaan], who gets assistance from the following community members who have committed to volunteering their free time:
+
+- [Busra Sengul][Busra Sengul]
+- [Emma Garland][Emma Garland]
+- [George Bidder][George Bidder]
+- [Jason Elkin][Jason Elkin]
+- [Laura Neto][Laura Neto]
```

Comment:
I think we are missing Kyle here 😁
Or did he drop off the team?

Reply:
Not sure how it got missing from the links 😱 
Will add him 👍